test/service/test_response_anvil.py

```diff
@@ -908,21 +908,37 @@ def test_entity_indices(self):
                     {
                         'activities': [
                             {
-                                'activity_type': ['Sequencing'],
-                                'assay_type': [None],
-                                'data_modality': [None]}],
+                                'activity_type': [
+                                    'Sequencing'
+                                ],
+                                'assay_type': [
+                                    None
+                                ],
+                                'data_modality': [
+                                    None
+                                ]
```

Comment:
I prefer the compressed "one-liner" syntax, as long as it fits the line length limits. If there is a inconsistency, I'd like the multiliners to be converted to oneliners.